qiita_pet/handlers/stats.py

```diff
@@ -18,10 +20,12 @@ def _get_stats(self, callback):
         # check if the key exists in redis
         lats = r_client.lrange('stats:sample_lats', 0, -1)
         longs = r_client.lrange('stats:sample_longs', 0, -1)
-        if not (lats or longs):
+        if not (lats and longs):
             # if we don't have them, then fetch from disk and add to the
             # redis server with a 24-hour expiration
-            lat_longs = get_lat_longs()
+            lat_longs = get_lat_longs(Portal(qiita_config.portal))
```

Comment:
It was "working" (i.e., not raising exceptions or anything), but the information that was being displayed was wrong.